app/build.gradle

```diff
@@ -40,7 +44,7 @@ dependencies {
     implementation "androidx.recyclerview:recyclerview:${versions.recyclerview}"
     implementation "androidx.constraintlayout:constraintlayout:${versions.constraint}"
     implementation "io.ktor:ktor-client-okhttp:${versions.ktor}"
-    implementation "io.ktor:ktor-client-gson:${versions.ktor}"
+    implementation "io.ktor:ktor-client-serialization-jvm:${versions.ktor}"
```

Comment:
just a doubt here, is it mandatory to build the serializer on the jvm?? 🤔 this would make the port to multiplatform harder

Reply:
Wow! It's true! I hadn't realized that! But fortunately the native one also exists! ^^ I changed it!